CardanoSharp.Wallet.Test/AddressTests.cs

```diff
@@ -125,11 +121,10 @@ public void VerifyRewardAddress(string mnemonic, string stakingAddr)
         /// <param name="path"></param>
         /// <param name="rootKey"></param>
         /// <returns></returns>
-        private (byte[], byte[]) getKeyPairFromPath(string path, (byte[], byte[]) rootKey)
+        private (PrivateKey, PublicKey) getKeyPairFromPath(string path, PrivateKey rootKey)
```

Comment:
Return KeyPair here (ie your suggested type)

Reply:
yeah I wrote this and thought about the Key (better name: KeyPair) type. Alright I get this added.